apps/site/assets/ts/schedule/components/line-diagram/__tests__/MergeTest.tsx

```diff
@@ -68,4 +70,14 @@ describe("Merge component", () => {
       wrapperNoBranches.exists("g.line-diagram-svg__merge path")
     ).not.toBeTruthy();
   });
+
+  it("handles bidirectional branching", () => {
+    const withBidi = mount(<Merges lineDiagram={bidiLineDiagram} />);
+    expect(
+      withBidi.findWhere(el => el.key() == "test station c-test station b-line")
+    ).toBeTruthy();
+    expect(
+      withBidi.findWhere(el => el.key() == "place-coecl-place-hymnl-line")
+    ).toBeTruthy();
+  });
```

Comment:
**suggestion:** We can expand on this later but maybe we can add a snapshot test here?